### PR TITLE
added batch effects reference

### PR DESCRIPTION
**Pull request by bdo311**
From #436

### CODE REVIEW

File: sections/04_study.md [hunk 1]
```diff
@@ -670,7 +670,9 @@ several percentage points of improvement compared to random forests or deep
 networks trained only on CpG or sequence information. Similar deep learning
 methods have been applied to impute low-resolution ChIP-seq signal from bulk
 tissue with great success, and they could easily be adapted to single cell data
-[@tag:Qin2017_onehot @tag:Koh2016_denoising].
+[@tag:Qin2017_onehot @tag:Koh2016_denoising]. Deep learning has also been useful
+for dealing with batch effects, which can severely affect single-cell data
```
Comment (agapow) [inline sections/04_study.md]: Well, batch effects can severely affect lots of areas, so the "single cell" qualification may not be needed.

Comment (cgreene) [inline sections/04_study.md]: Reference for dealing with batch effects? Might be nice to guide the reader.

Comment (cgreene) [inline sections/04_study.md]: Ahh - after another glance - I see what you've done here. I was interpreting the reference as related to the ", which" clause.

Comment (cgreene) [inline sections/04_study.md]: I agree with @agapow now - I think it's clearer without that clause.

**Comment by agitter** [comment]
@bdo311 Can you please remove the TODO at the end of the single cell section? That was a reminder to add this paper.

**Pull request merged**